docs/source/release/v6.13.0/Framework/Algorithms/Removed/38793.rst

```diff
@@ -0,0 +1 @@
+- Removed the obsolete algorithm `SaveISISNexus`.
```

Comment:
Normally we would mark algorithms as deprecated for at least two releases before finally removing them as per https://docs.mantidproject.org/nightly/deprecation.html . Do we perhaps need to be clear here why this is not being done?

Reply:
Suggestion for better phrasing (please comment):

```rst
- Removed the obsolete algorithm ``SaveISISNexus``. It is being removed before the normal comment period rather than undergoing extensive changes to accommodate the consolidation of nexus APIs in mantid.
```

.... and this should be double-backticks because it is rst rather than markdown